modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/src/main/resources/site-initializer/fragments/group/evp/fragments/slider-evp/configuration.json

```
@@ -0,0 +1,22 @@
+{
+	"fieldSets": [
+		{
+			"fields": [
+				{
+					"dataType": "int",
+					"defaultValue": "3",
+					"label": "number-of-slides",
```

Please, change the label name here.
```suggestion
					"label": "Number of slides",
```